posthog/api/session_recording.py

```diff
@@ -201,3 +214,18 @@ def snapshots(self, request: request.Request, **kwargs):
                 }
             }
         )
+
+    # Returns metadata given a list of session recording ids
+    @action(methods=["GET"], detail=False)
```

Comment:
Not sure about this approach tbh. We are passing in the same query as for the recordings list and hoping we get the same results but that might not be the case. 

As we only list a certain number of sessions, I wonder if we couldn't do this differently and have a simple request passing in session_ids and returning with their metadata? That way it is pretty much a static (and highly cacheable) endpoint and it doesn't matter if new recordings come in that changes the list.

For now I'll probably merge as it is given this is all behind a feature flag anyways but we should perhaps revisit